### ISSUE TITLE
Issue: mehrere Warenkorb-Rabatte ohne Zwischensumme

### ISSUE THREAD

**Issue by paulhaem** [isotope/core#1705]
Hallo,
folgendes Problem:
Ein Kunde verwendet mehrere Rabatte die sich auf den kompletten Warenkorb beziehen. Ihm ist dabei aufgefallen, dass die Rabatte ohne Zwischensumme berechnet werden. Das heißt, alle Prozente werden von einer Zwischensumme gerechnet und dann zusammen abgezogen. Normal wäre es doch, dass pro Rabatt eine Zwischensumme gebildet werden müsste, oder etwa nicht? Es macht bei den Bestellsummen teilweise recht große Unterschiede, je nachdem wie man das berechnet. Ist mir bisher persönlich auch noch nie so aufgefallen. Ist das so gewollt?

**Comment by aschempp** [comment]
Bisher ist es in Isotope so, dass alle "Zuschläge" oder "Abschläge" sich immer auf das Subtotal beziehen. Beispielsweise enthält ein Zuschlag für Zahlungskosten auch nie die Versandkosten. In der aktuellen Version lässt sich das aus Design-Gründen nicht ändern.

**Comment by Irontale** [comment]
Kann ich nachvollziehen. Wenn wir aber z.B. von einem Warenkorb-Wert von 100€ ausgehen, und du gibst dem Kunden einen 50% Skonto und einen Aktions-Rabatt von 50%, landest du bei einem Bestellwert von 0€, nach aktuellem Stand der Dinge, anstatt 25€ was richtig wäre. Es fehlen sozusagen die Zwischensummen ... :/

100€ - 50€ (50%) - 50€ (50%) = 0€
100€ - 50€ (50%) = 50€ - 25% (50%) = 25€

**Comment by aschempp** [comment]
Da hast du sicherlich recht. Nur…
> In der aktuellen Version lässt sich das aus Design-Gründen nicht ändern.

**Comment by paulhaem** [comment]
Hat sich mittlerweile in dieser Richtung etwas getan?

**Comment by aschempp** [comment]
Nein, bisher nicht. Allerdings hatte ich eine Idee wie sich dies in der aktuellen Version möglicherweise doch machen lässt, ich öffne daher das Thema nochmals.
